eeps/eep-0007.md

```diff
@@ -485,12 +458,12 @@ Types:
 - `Port = port()`
 
 - `CFunc = c_func_name() | func_index()
-    | tagged_func_name()_ | `tagged_func_index()`
```

Comment:
An actual error, though I think some of these you might've picked up already before, @garazdawi. I still did them here, for the sake of completeness.